### PR TITLE
DOCSP-37598-mongodb-shell-2.20-release

### PR DESCRIPTION
**Pull request by mmaville-mdb**
## DESCRIPTION
Release Notes for MongoDB Shell 2.2.0. Please note this was originally submitted as 2.1.6 and upreved to 2.2.0 after adding some additional work items.

## STAGING
https://preview-mongodbmmavillemdb.gatsbyjs.io/mongodb-shell/DOCSP-37598-mongodb-shell-2.20-release/changelog/

## JIRA
https://jira.mongodb.org/browse/DOCSP-37598

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65fb47d36fc5f048506ae54f

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by kennethdyer — CHANGES_REQUESTED** [review]
Thanks for this work @mmaville-mdb, I left some comments for your consideration.

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,57 @@ Release Notes
...
    :class: singlecol
 
+v2.2.0
+------
+
+*Released March 11, 2024*
+
+Performance Improvements
+
+- :issue:`MONGOSH-1605` – ``mongosh`` now makes use of Node.js startup
```
[the remark this answers is not in the document]
Comment (mmaville-mdb) [reply source/changelog.txt]: Simplified to "uses" since "now uses... again" felt a bit clunky after seeing the edit.

**Review by kennethdyer — APPROVED** [review]
Thanks @mmaville-mdb, LGTM

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,71 @@ Release Notes
...
+- :issue:`MONGOSH-1738` - ``require`` not working in script mode.
+
+v2.2.0
+------
+
+*Released March 11, 2024*
+
+.. important::
+     This release is affected by a bug, fixed in 2.2.1, where ``require('<module>')`` 
+     caused an error in script mode.
```
Comment (kennethdyer) [inline source/changelog.txt:32]: I don't see a build error for this and the rendering seems ballpark right, so maybe Snooty is making some adjustments, but the indentation looks wrong.

There should be a blank line after the directive.   Lines under the directive should have three space indents to bring them in line with the first letter of the directive type.

File: source/changelog.txt [hunk 3]
```diff
@@ -12,6 +12,71 @@ Release Notes
...
+script mode:
+
+- :issue:`MONGOSH-1738` - ``require`` not working in script mode.
+
+v2.2.0
+------
+
+*Released March 11, 2024*
+
+.. important::
```
Comment (kennethdyer) [inline source/changelog.txt:30]: [Suggestion]

Given that we're highlighting a bug, the `warning` direcive may be more appropriate.

**Review by kennethdyer — COMMENTED** [review]
Thanks @mmaville-mdb.  Left a couple comments for your consideration.

**Comment by mmaville-mdb** [comment]
@GaurabAryal gave approval via Slack on 3/20/24.

**Pull request merged**